src/cpip-pack

```diff
@@ -171,16 +171,31 @@ install_pip()
     fi
 }
 
+get_poetry_major_version()
+{
+    poetry_major_version=$(poetry --version | awk '{print $3}' | cut -f1 -d '.')
```

Comment:
double quotes around this for future safety

Reply:
ditto for the line below

Reply:
Done.